e2e/README.md

```diff
@@ -0,0 +1,12 @@
+# E2E Tests
```

Comment:
Should we quickly mention this one in the main README? https://github.com/bpmn-io/form-js#build-and-run

So it's easier to find on first setup.